collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Description: alarms netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from json import loads
+
+from bases.FrameworkServices.UrlService import UrlService
+
+priority = 25
```

Comment:
I am not sure about this priority, it makes `Alarms` section appear on the top, above `System Overview`.

@cakrit @dim08 (see https://github.com/andrewm4894/netdata/blob/alarms-collector/collectors/python.d.plugin/alarms/README.md#charts)

Reply:
I have set it to be `200200` so appears below "Netdata Monitoring". There is also an argument for having it appear within "Netdata Monitoring" as an "alarms" section perhaps. If we wanted, is that possible to do with a collector?

Reply:
> I have set it to be 200200

That is another edge case, i think nothing should appear below "Netdata Monitoring".

Reply:
Why not just have default priority? Why do you think default is not good?